src/static/styles/App.scss

```diff
@@ -175,20 +175,67 @@ hr {
 	}
 }
 
+//fix home: use this same techine on yellow part, previous color was .bg-secondary-dark
+.smallbluebox{
+	background-color: $secondary-dark-theme;
+	@include md{
+		background-color: transparent;
+	}
+}
+
+.bigbluebox{
+	background-color: transparent;
+	@include md{
+		background-color: $secondary-dark-theme;
+	}
+}
+
+.smallyellowbox{
+	background-color: $secondary-theme-dark;
+	@include md{
+		background-color: transparent;
+	}
+}
+
+.bigyellowbox{
+	background-color: transparent;
+	@include md{
+		background-color: $secondary-theme-dark;
+	}
+}
+
+//the culprit, fix these font sizes to fix issues. also change border width
 .hero-header1 {
 	font-family: Raleway, sans-serif;
 	font-weight: bold;
-	font-size: 4.4625rem;
+	font-size: 2rem;
 	line-height: 1.2;
+	@include sm {
+		font-size: 3.5rem;
```

Comment:
it would be good to standardize a sm, md, and lg font-size at the top of the CSS class